test/plugin_functional/plugins/core_http/public/plugin.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { Plugin, CoreSetup } from 'kibana/public';
+
+export class CoreHttpPlugin implements Plugin<CoreHttpPluginSetup, CoreHttpPluginStart> {
+  public setup({ http }: CoreSetup, deps: {}) {
+    const tryRequestCancellation = async () => {
+      const abortController = new AbortController();
+      abortController.abort();
+
+      try {
+        await http.get('/api/core_http/never_reply', { signal: abortController.signal });
```

Comment:
does it even perform a request if you pass the aborted signal? Maybe we should call `abort()` after the request has been sent?

Reply:
The `XMLHttpRequest` object is created, but you're right, I'm not sure it goes further than readyState 0 when aborting before the call. In doubt, adapted in https://github.com/elastic/kibana/pull/97086/commits/86a6dd25a54625930e1149ada845a36a768b7d13